index.html

```diff
@@ -17,170 +17,189 @@
                 </div>
             </div>
         </div>
     </header>
 
     <section class="features" id="features">
-        <h2 class="section_header features__header">features</h2>
+        <h2 class="features__header section-header">features</h2>
         <div class="wrapper">
-            <div class="features__content">
-                <div class="features__content--card">
-                    <div class="features__content--card-circle"><img src="./images/responsive.svg" alt=""></div>
-                    <span class="features__content--card-title">fully responsive</span>
-                    <p class="features__content--card-txt">looks amazing on any device:
+            <ul class="features__cards">
+                <li class="features__card">
+                    <figure class="features__circle"><img src="./images/responsive.svg" alt=""></figure>
+                    <span class="features__title">fully responsive</span>
+                    <p class="features__txt">looks amazing on any device:
                         smarphone, tablet, laptop and desktop.
                     </p>
-                </div>
-                <div class="features__content--card">
-                    <div class="features__content--card-circle"><img src="./images/customizable.svg" alt=""></div>
-                    <span class="features__content--card-title">customizable</span>
-                    <p class="features__content--card-txt">Change the colors, pictures or any of the sections to suit
+                </li>
+                <li class="features__card">
+                    <figure class="features__circle"><img src="./images/customizable.svg" alt=""></figure>
+                    <span class="features__title">customizable</span>
+                    <p class="features__txt">Change the colors, pictures or any of the sections to suit
                         your needs.
                     </p>
-                </div>
-                <div class="features__content--card">
-                    <div class="features__content--card-circle"><img src="./images/design.svg" alt=""></div>
-                    <span class="features__content--card-title">slick and beautiful design</span>
-                    <p class="features__content--card-txt">Trendy and fresh design, fits any website
+                </li>
+                <li class="features__card">
+                    <figure class="features__circle"><img src="./images/design.svg" alt=""></figure>
+                    <span class="features__title">slick and beautiful design</span>
+                    <p class="features__txt">Trendy and fresh design, fits any website
                     </p>
-                </div>
-            </div>
+                </li>
+            </ul>
         </div>
     </section>
 
     <section class="hero-img">
-        <div class="hero-img-bg"></div>
+        <figure class="hero-img__bg"></figure>
```

Comment:
Raczej przyjęło się, że ten znacznik powinien mieć zawartość.
Jeśli jest to tło to raczej `figure` nie używamy :)